templates/server/puppet.conf.main.erb

```diff
@@ -0,0 +1,13 @@
+## Server config
+
+    # Puppet 3.0.x requires this in both [main] and [master] - harmless on agents
+    autosign       = <%= scope.lookupvar('::puppet::autosign') %>
+    reports     = <%= scope.lookupvar('::puppet::server::reports') %>
```

Comment:
Alignment is off. Also please leave an empty line in between. It now looks like the comment is also for reports.